Showcase-iOSTests/Dashboard/Presenter/HomePresentableTests.swift

```diff
@@ -33,33 +34,40 @@ class HomePresentableTests: XCTestCase {
         homePresenter.firebaseStorage = mockFirebaseStorage
         homePresenter.signOutInteractor = mockSignOutInteractor
         homePresenter.userDefaults = mockUserDefaults
-        systemUnderTest = homePresenter
+        homePresenter.analyticManager = mockAnalyticsManager
+        presenterUnderTest = homePresenter
```

Comment:
why did you rename this to presenterUnderTest? systemUnder test is a more universal term for unit testing so that we know what the class we are testing. Please revert the renaming of systemUnderTest

Reply:
done